itest/rust/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = ["dep:serde", "dep:serde_json", "godot/serde"]
 godot = { path = "../../godot", default-features = false, features = ["__trace"] }
 serde = { version = "1", features = ["derive"], optional = true }
 serde_json = { version = "1.0", optional = true }
+pin-project-lite = { version = "0.2" }
```

Comment:
This looks indeed relatively lightweight, [70kB according to lib.rs](https://lib.rs/crates/pin-project-lite) :slightly_smiling_face: